params/chromium-params.md

```diff
@@ -29,6 +29,8 @@ Chromium's implementation assigns the following values:
 | [Max attributions per rate-limit window][] | [100][max attributions per rate-limit window value] |
 | [Allowed aggregatable budget per source][] | [65536][allowed aggregatable budget per source value] |
 | [Randomized aggregatable report delay][] | [10 minutes][randomized aggregatable report delay value] |
+| [Max information gain for navigations][] | [11.46 bits][max information gain for navigations value] |
+| [Max information gain for events][] | [6.5 bits][max information gain for events value] |
```

Comment:
It might be clearer to say `for navigation sources` and `for event sources`.

Reply:
edited